test/libsolidity/ABIJson/event_emitted_from_foreign_contract.sol

```diff
@@ -0,0 +1,96 @@
+contract C {
+    event E(address indexed sender);
+}
+contract E {
+    event G(uint256 value);
+}
+contract D {
+    function test(address sender) public {
```

Comment:
Please add an event called `E` in contract `C` as well.

Also, would not hurt if contract `E` was an interface.

Reply:
You mean add an event `E` to contract `E`? I'll change the contract/event names to make this easier to read.

Reply:
Ah, sorry. I meant `D`. I.e. put an event with the same signature both in `C` and in `D`.